plugins/sub_plugins/validate/jsonschema.py

```diff
@@ -153,30 +207,43 @@ def _validate_jsonschema(self):
         error_messages = None
 
         draft = self._get_sub_plugin_options("draft")
+        check_format = self._get_sub_plugin_options("check_format")
         error_messages = []
 
         for criteria in self._criteria:
-            if draft == "draft3":
-                validator = jsonschema.Draft3Validator(
-                    criteria,
-                    format_checker=jsonschema.draft3_format_checker,
-                )
-            elif draft == "draft4":
-                validator = jsonschema.Draft4Validator(
-                    criteria,
-                    format_checker=jsonschema.draft4_format_checker,
-                )
-            elif draft == "draft6":
-                validator = jsonschema.Draft6Validator(
-                    criteria,
-                    format_checker=jsonschema.draft6_format_checker,
+            format_checker = None
+            validator_class = None
+            try:
+                validator_class = self._JSONSCHEMA_DRAFTS[draft]["validator"]
+            except KeyError:
+                display.vvv(
+                    'No jsonschema validator for "{draft}", falling back to autodetection.'.format(
```

Comment:
We don't really need a rigid mapping of draft to minimum jsonschema version, (although it would probably help for clarity and probably for sanity checking) but mentioning that a missing draft can be found in a newer version of jsonschema would be a nicer response than effectively ignoring user input